src/d3d12/d3d12_structs.hpp

```diff
@@ -184,6 +196,10 @@ namespace wr::d3d12
 	{
 		desc::RootSignatureDesc m_create_info;
 		ID3D12RootSignature* m_native;
+
+		uint32_t m_num_descriptors_per_table[32];
```

Comment:
please use std::uint32_t to be consistent.

Reply:
👍